apps/met-anomalycor/server/main.js

```diff
@@ -449,8 +449,17 @@ const doCurveParams = function () {
         );
     }
 
-    var defaultGroup = (Object.keys(dbGroupMap).indexOf("NO GROUP") !== -1) ? "NO GROUP" : Object.keys(dbGroupMap)[0];
-    var defaultDB = (dbGroupMap[defaultGroup].indexOf("mv_gsl_global_g2g") !== -1) ? "mv_gsl_global_g2g" : dbGroupMap[defaultGroup][0];
+    var defaultGroup;
+    var defaultDB;
+    if (Object.keys(dbGroupMap).indexOf("NOAA NCEP") !== -1 && dbGroupMap["NOAA NCEP"].indexOf("mv_gfs_grid2grid_metplus") !== -1) {
+        // special default parameters for the NWS METexpress
+        defaultGroup = "NOAA NCEP";
+        defaultDB = "mv_gfs_grid2grid_metplus";
+    } else {
+        // default to the GSL g2g database if it exists, otherwise default to whatever is first alphabetically
+        defaultGroup = (Object.keys(dbGroupMap).indexOf("NO GROUP") !== -1) ? "NO GROUP" : Object.keys(dbGroupMap)[0];
+        defaultDB = (dbGroupMap[defaultGroup].indexOf("mv_gsl_global_g2g") !== -1) ? "mv_gsl_global_g2g" : dbGroupMap[defaultGroup][0];
+    }
```

Comment:
Instead of hardcoding these defaults would this be something that would be easier to do in a config file? Then whenever the defaults change we could just update the config instead of having to update MATS itself.

If we did want to have a default, (e.g. - to match a static dev/test database) we could leave that in here and then set the value to whatever's in the config file if it's present.

I think it's okay for the merge to go forward as-is but thought I'd bring this up.

Reply:
We can indeed move the defaults to a settings file. I've been trying not to overload the settings files, as they have to be created by someone installing METexpress out of the box (and that someone might be a sysadmin who doesn't know anything about the database), but it would be nice to not have to release an update every time we want to modify the defaults. I'll test this on dev-localhost settings and see how onerous it is.